### ISSUE TITLE
[stable/fluentd] Duplicate pvc template

### ISSUE THREAD

**Issue by duanshiqiang** [helm/charts#17540]
**Describe the bug**
Inside fluentd chart templates, there is one `pvc.yml` file as well as one `pvc.yaml` file. I think it's a mistake.

**Comment by chicken231** [comment]
It sure seems like it:

```
helm install --name fluentd stable/fluentd --version 2.1.3
Error: release fluentd failed: persistentvolumeclaims "fluentd" already exists
```

**Comment by procyclinsur** [comment]
Seems like this was fixed 5 days ago here #19131
